relay.go

```diff
@@ -67,14 +71,16 @@ type relayItems struct {
 	sync.RWMutex
 
 	logger Logger
+	wheel  *timers.Wheel
 	tombs  uint64
 	items  map[uint32]relayItem
 }
 
-func newRelayItems(logger Logger) *relayItems {
+func newRelayItems(wheel *timers.Wheel, logger Logger) *relayItems {
```

Comment:
super nit: ordering of args is `wheel, logger` but ordering of fields in struct and initializer is `logger, wheel`